source/precalculus/source/08-TE/01.ptx

```diff
@@ -12,6 +12,1187 @@
     <subsection>
     <title>Activities</title>
 
+    <remark>
+      Pythagorean Identities, as we have seen in a previous section, are derived from the Pythagorean Theorem <m> \left( a^2+b^2=c^2 \right) </m>. Pythagorean Identities are used in solving many trigonometric problems where one trigonometric ratio is given and we are expected to find the other trigonometric ratios. The most commonly used Pythagorean Identity is: <me>\sin^2\theta + \cos^2\theta = 1</me>. The next two activities will lead us to find the other two Pythagorean Identities.
```

Comment:
We should link to the section and ideally the activity/observation/etc. this is done in (though I can't find where this is done previously).  

If needed, the subsequent activity can easily be rewritten to derive this.

Reply:
According to our outline, we wanted to cover the first identity in TR5 (which has not been written yet). I didn't want to link yet until it was written.